lib/core/widgets/back_button.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:deus/statics/styles.dart';
+import 'package:flutter/material.dart';
+
+class BackButtonWithText extends StatelessWidget {
```

Comment:
Mir ist aufgefallen, dass du den Button meist abgeschnitten hast. 
![grafik](https://user-images.githubusercontent.com/54584421/108824966-b07cb480-75c2-11eb-964e-51705c0805ec.png)

Um das zu vermeiden, 
- [ ] bette ihn bzw. das ganze Scaffold in ein SafeArea ein (auf den Screens, nicht in dem Widget)

Reply:
Das es so abgeschnitten ist liegt daran, dass es etwas runtergescrolled war